Packs/GoogleCloudCompute/Integrations/GoogleCloudCompute/GoogleCloudCompute.yml

```diff
@@ -3,10 +3,10 @@ commonfields:
   id: Google Cloud Compute
   version: -1
 configuration:
-- display: Service Account Private Key file contents (JSON)
-  name: service
-  required: true
-  type: 4
+- display: add Service Account Private Key file contents (JSON) to "Password"
+  name: credentials
+  required: false
```

Comment:
why not `required`?

Reply:
by accident. I meant for it to be required. I though I did it :)